langfuse/model.py

```diff
@@ -76,7 +76,8 @@ def _get_langchain_prompt_string(content: str):
         return re.sub(r"\{\{(.*?)\}\}", r"{\g<1>}", content)
 
     @staticmethod
-    def _compile_template_string(content: str, **kwargs) -> str:
+    def _compile_template_string(*args, **kwargs) -> str:
```

Comment:
Not sure at all about the way I fixed this since I'm new to Python, I let you handle it if there's a better way :smiley: